Documentation/compatibility/wpf-TextBox-PasswordBox-text-selection-does-not-follow-system-colors.md

```diff
@@ -0,0 +1,40 @@
+## WPF TextBox/PasswordBox Text Selection Does Not Follow System Colors
+
+### Scope
+Edge
+
+### Version Introduced
+4.7.2
+
+### Source Analyzer Status
+NotPlanned
+
+### Change Description
+In .NET Framework 4.7.1 and earlier versions, WPF `System.Windows.Controls.TextBox` and `System.Windows.Controls.PasswordBox` could only render a text selection in the Adorner layer.
+In some system themes this would occlude text, making it hard to read.  In .NET Framework 4.7.2 and later, developers have an option of enabling a non-Adorner based selection rendering 
+scheme that alleviates this issue.
+
+- [X] Quirked
+- [ ] Build-time break
+
+### Recommended Action
```

Comment:
Here again, it's best to break these up into two paragraphs with two code blocks, assuming that apps that target versions earlier than 4.7.2 but are running on 4.7.2 can take advantage of the change.

Reply:
I changed it to only give directions on how to enabled the non-adorner-based selection.  Since the default is always adorner-based, regardless of version.